tests/test_ensemble_distribution.py

```diff
@@ -0,0 +1,52 @@
+import copy
+
+import numpy as np
+import pandas as pd
+import pytest
+from conftest import assert_equal
+
+from risk_distributions.formatting import Parameter, Parameters
+from risk_distributions.risk_distributions import EnsembleDistribution
+
+weights_base = {
+    "betasr": 1 / 12,
```

Comment:
Wouldn't it be better if the weights weren't all equal? This would prove that we're not somehow mixing them up.